config/services/cratedb.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<service>
+  <short>ceph-mon</short>
+  <description>CrateDB is a distributed SQL database management system that integrates a fully searchable document oriented data store.</description>
+  <port protocol="tcp" port="4200"/>
+  <port protocol="tcp" port="4300"/>
+  <port protocol="tcp" port="5432"/>
```

Comment:
I believe port `5432` should be moved to a separate service file such as `cratedb-postgresql.xml` or even better, mention in the description to also use the `postgresql` service if PostgreSQL wire support is enabled.

Reply:
If CrateDB requires this port by default, then we should include it here. Since it's the same port, we could just include the postgresql service, e.g. `<include service="postgresql" />`.

Reply:
> If CrateDB requires this port by default, then we should include it here. Since it's the same port, we could just include the postgresql service, e.g. `<include service="postgresql" />`.

It's undocumented but the postgresql wire protocol is enabled by default, so the `postgresql` service file should be include as you mentioned.